sage-ios/SAGE/SAGE/ProfileView.swift

```diff
@@ -0,0 +1,224 @@
+//
+//  ProfileView.swift
+//  SAGE
+//
+//  Created by Erica Yin on 11/21/15.
+//  Copyright © 2015 Cal Blueprint. All rights reserved.
+//
+
+import Foundation
+
+class ProfileView: UIView {
+    
+    let profileContent = UIView()
+    let header = UIView()
+    let profileUserImg = UIImageView()
```

Comment:
make sure in the ProfileView deinit that you cancel the image request operation for the profile image